website/content/gateway/api_reference/resources/resources.md

```diff
@@ -5,11 +5,13 @@ anchor: "resources"
 weight: 100
 ---
 ## Resources
-Our API offers six different resources:
+Our API offers eight different resources:
 
 - [Authorizations](#authorizations)
 - [Captures](#captures)
 - [Refunds](#refunds)
```

Comment:
Should we rename these to "Capture refunds" to be consistent?

Reply:
At first I wasn't in doubt, however, now I am.

Ordinary refunds hang on an authorization, not on a capture. Of course, there is a relationship to an authorization's capture(s), as one or more captures must have taken place before a refund can be made.

Try to taste this sentence from the current Refunds section:

> You can make multiple refunds for an authorization transaction.

Changing "Refunds" to "Capture refunds" will not break things in any deep way, but the heading suggests that one POSTs on `/captures/:id/refunds` which is not the case.

@ct-clearhaus WDYT?

Reply:
"Authorization refunds" ... I don't like this one because one can in fact (towards schemes) authorize refunds, and this would be confusing.

"DMS refunds"? We use "debit" as the terminology for "SMS" ... what's our terminology for "DMS"? We have none. That's the culprit of our naming problem, I think.

Reply:
I don't like "Authorization refunds" either, and I think it's a big step to introduce SMS and DMS :thinking: 

As a first stab I've pushed https://github.com/clearhaus/gateway-api-docs/pull/146/commits/ab4a3b0edccd603e9acd3560d4eef078f4f693e9.

Reply:
I'll allow myself to resolve this thread. We can pick it up later in another PR.